pkg/plugin/v2/scaffolds/internal/templates/manager/config.go

```diff
@@ -67,6 +67,8 @@ spec:
       labels:
         control-plane: controller-manager
     spec:
+      securityContext:
+        runAsUser: 1337
```

Comment:
Could you please do it for v3 only and as it was done for the proxy in https://github.com/brancz/kube-rbac-proxy/pull/86? We need to update the image as well. So the change will be:
- in /pkg/plugin/v3/scaffolds/internal/templates/manager/config.go
- in v3/scaffolds/internal/templates/dockerfile.go
And then, see that you need to run `make generate` to update the samples.

Reply:
Okay, updated the PR